runtime/centrifuge/src/lib.rs

```diff
@@ -288,7 +311,39 @@ impl pallet_restricted_tokens::Config for Runtime {
 	type PreFungiblesTransfer = cfg_traits::Always;
 	type PreReservableCurrency = cfg_traits::Always;
 	type RuntimeEvent = RuntimeEvent;
-	type WeightInfo = weights::pallet_restricted_tokens::WeightInfo<Self>;
+	type WeightInfo = ();
```

Comment:
I'm pretty sure we don't want this. I think the calls are exposed.

Default implementations have 0 weights.

Reply:
Done. Needed to wait for next benchmark